speaker_verification/deep_speaker/audio.py

```diff
@@ -8,14 +8,24 @@
 
 import logging
 import os
+from datetime import date
 from glob import glob
 from random import choice
 
 import librosa
 import numpy as np
 from python_speech_features import fbank
 
-logger = logging.getLogger(__name__)
+from speaker_verification.utils.logger import SpeakerVerificationLogger
+
+today = date.today()
+file_path = os.path.dirname(os.path.abspath(__file__))
+logger = SpeakerVerificationLogger(
+    name=__file__,
+    log_file=f'{file_path}/../logs/logs-{today.strftime("%d-%m-%Y")}.log'
```

Comment:
Minor but maybe we could omit `logs-` from the logfile name since its contained within the `logs` dir and uses the `.log` extension.
```suggestion
    log_file=f'{file_path}/../logs/{today.strftime("%d-%m-%Y")}.log'
```

Reply:
Sure!